### ISSUE TITLE
Save page with a keyboard key combination like Ctrl + S

### ISSUE THREAD

**Issue by WebAtelier47** [gantry/gantry5#2268]
Hello, 
Just an idea for gantry 5 on **particule default page** and **page settings** add the possibility to save the page with a keyboard key combination like Ctrl + S, this will be very usefull. Thanks.

**Comment by N8Solutions** [comment]
I agree that having a key combination to save the settings when modifying the Gantry template would be very helpful but it would need to be something different than `ctrl+s` as that is already used by default in most browsers. I'm sure a suitable alternative could be found though.
